tests/integration/test_slice.py

```diff
@@ -49,7 +49,9 @@ class Environment:
         pip=["gym[atari]"],
     ),
     "pytorch": lambda: Environment(
```

Comment:
weird that the diff is showing up... maybe you haven't rebased yet with main (I thought this was merged already)?